community/settings.py

```diff
@@ -131,3 +132,4 @@
 MEDIA_URL = '/media/'
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 STATICFILES_DIRS = ['static/']
+OH_TOKEN = ''
```

Comment:
you need to get the value from the environment variable `OH_TOKEN`.  Use `gci.client.get_api_key('OH')`